docs: clarify that support schedule is subject to change (#40770)
angular/angular: aio/content/guide/releases.md

<div class="alert is-helpful">

> Disclaimer: Dates are offered as general guidance and will be adjusted by us when necessary to
> ensure delivery of a high-quality platform.

Reviewer: ```suggestion
Dates are offered as general guidance and subject to change.
```

{@a support}
## Support policy and schedule

<div class="alert is-helpful">

> Disclaimer: Dates are offered as general guidance and will be adjusted by us when necessary to
> ensure delivery of a high-quality platform.

Reviewer: ```suggestion
Dates are offered as general guidance and subject to change.
```

</div>

> All of our major releases are typically supported for 18 months.

Reviewer: ```suggestion
All major releases are typically supported for 18 months.
```
